systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2IsolatedST.java

```diff
@@ -1633,6 +1635,39 @@ void testKMM2RollAfterSecretsCertsUpdateTLS(ExtensionContext extensionContext) {
         LOGGER.info("Messages successfully mirrored");
     }
 
+    @ParallelNamespaceTest
+    void testMirrorMaker2RackAwareness(ExtensionContext extensionContext) {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(INFRA_NAMESPACE, extensionContext);
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        String kafkaClusterSourceName = clusterName + "-source";
+        String kafkaClusterTargetName = clusterName + "-target";
```

Comment:
You can use here `TestStorage` as well -> for Kafka names then use just `testStorage.getClusterName() + "-source"/"-target"`

Reply:
OK. Thanks.